test/lint.ts

```diff
@@ -54,8 +54,12 @@ const loadAndCheckFiles = async (...files: string[]): Promise<DataType> => {
         full: path.relative(process.cwd(), file),
         category: '',
       };
-      if (filePath.full.includes(path.sep)) {
-        filePath.category = filePath.full.split(path.sep)[0];
+      if (path.sep !== '/') {
+        // Normalize file paths for Windows users
+        filePath.full = filePath.full.replace(path.sep, '/');
```

Comment:
@queengooborg I added some crappy logging . It seems to be showing that this only replaces the first separator, and you have several.

```
Before replace: webextensions\manifest\storage.json 
After replace: webextensions/manifest\storage.json 
```

Could this be the cause? If not, add logging where you need and I'll run this for you.

Reply:
Ack, right, I forgot that JavaScript's `.replace()` function is dumb... 🤦‍♀️

Reply:
Yeah, lets move to Python for all of this :-)